2019/2019-10-16-meeting.md

```diff
@@ -0,0 +1,35 @@
+# SR KTC Meeting 2019-10-02
+
+Date: Wednesday 2nd October 2019, 19:30
+
+Present: Kajetan Champlewski, Kier Davis, Rob Gilton, Dan Trickey
+
+## Minutes
+
+* Kickstart kit prep
+    * all packed:
+        * Southampton kits are in Tom's garage
+        * London and Cambridge kits are in the container; some competition team volunteers have agreed to take responsibility for shipping these to the respective venues
+        * Tom, Rob and Kier each have a spare kit box in their possession for the Southampton, Cambridge and London kickstarts respectively
+* Getting software ready for kickstart
+    * Updating the overlay (robot API, maybe a libkoki fix)
+        * Rob is already working this
+    * Creating the marker PDFs
+    * Updating the docs (types of marker)
+    * Updating the dummy API
+    * Updating the simulator
+        * There's been a miscommunication between the CT and KT here: the CT are currently assuming that it exists (it's in the kickstart microgames) but the KT isn't currently planning to ship it and doesn't currently have the resources to do so in time for kickstart.
+        * Motion: recommend to the competition team that we will not be providing a simulator for the SR2020 game in time for SR2020 kickstart, but they are welcome to use the simulator from SR2019.
+            * Motion passed.
+    * Instead of a meeting next week, we'll have a "hack evening" to work on these jobs.
+* Setting up issue creation bot
+    * Wont fix - github project workflow makes this redundant
+* Kit logistics responsibility
+    * Lots of miscommunication here
+    * Agreed that in the long term the kit team should take responsibility for shipping all kits to/from teams/kickstart venues
+    * However it's pretty infeasible for us to handle it for this coming Kickstart, hence the (informal) agreement with the competition team
```

Comment:
There is no such agreement.

Reply:
Please contact us via email if you wish to discuss anything mentioned in our minutes.